utils/build/docker/java/spring-boot.Dockerfile

```diff
@@ -28,5 +28,6 @@ RUN chmod +x /app/app.sh
 ENV DD_TRACE_HEADER_TAGS='user-agent:http.request.headers.user-agent'
 ENV APP_EXTRA_ARGS="--server.port=7777"
 ENV DD_DATA_STREAMS_ENABLED=true
+ENV DD_TRACE_OTEL_ENABLED=true
```

Comment:
This env var must be set at the scenario level : https://github.com/DataDog/system-tests/blob/main/utils/_context/_scenarios.py#L1103

Reply:
Thanks, I wound have guess this part.
I pushed the changes and hope it will work in CI (still can't run things locally).